src/System.Drawing.Common/src/System/Drawing/Imaging/EncoderParameters.cs

```diff
@@ -35,7 +35,7 @@ public EncoderParameter[] Param
         /// <summary>
         /// Copy the EncoderParameters data into a chunk of memory to be consumed by native GDI+ code.
         ///
-        /// We need to marshal the EncoderParameters info from/to native GDI+ ourselve since the definition of the managed/unmanaged classes
+        /// We need to marshal the EncoderParameters info from/to native GDI+ ourself since the definition of the managed/unmanaged classes
```

Comment:
*Ourself* with such uses of *we* is valid, but *ourselves* is too, and perhaps more usual.